ARM-open/arm-open.github.io
# feat: yarn.lock file for yarn usage on heroku

# What this is

A yarn.lock file so you can use the amazing yarn pm on heroku!
## Status
**READY**

## Description
- Speed up the deployment time with yarn over that annoying npm
- Saves you time, and time is something that is most previous

## Steps to Test or Reproduce
- Just add yarn.lock to your heroku git and then push it into heroku
- Make sure you force git add it, because yarn.lock is contained in most .gitignore files

## Impacted Areas in Application
- None
